### PR TITLE
[DOC] update TVM EP docs

### PR DESCRIPTION
**Pull request by vvchernov**
Update TVM docs after https://github.com/microsoft/onnxruntime/pull/10211 and https://github.com/microsoft/onnxruntime/pull/10260 were merged

@tmoreau89 please see

### CODE REVIEW

**Comment by tmoreau89** [comment]
Thank you @vvchernov! @xadupre - are you able to help review this one?

File: docs/execution-providers/TVM-ExecutionProvider.md [hunk 1]
```diff
@@ -74,22 +47,39 @@ sudo apt-get install kitware-archive-keyring
...
+python3 -m pip install $whl_path
 ```
 
-Build the python API for ONNX Runtime instead of using the standard package:
+Package for TVM EP:
 ```bash
 cd <path_to_onnx_runtime>
-pip3 uninstall onnxruntime onnxruntime-stvm -y
-whl_path=$(find ./build/Linux/Release/dist -name "*.whl")
+python3 -m pip3 uninstall onnxruntime onnxruntime-tvm -y
```
Comment (xadupre) [inline docs/execution-providers/TVM-ExecutionProvider.md:74]: -m pip uninstall

Comment (vvchernov) [reply docs/execution-providers/TVM-ExecutionProvider.md]: Thanks @xadupre!

**Review by xadupre — APPROVED** [review]

**Review by xadupre — APPROVED** [review]

**Comment by tmoreau89** [comment]
Hi @xadupre - please merge if you have the permissions thank you!

**Comment by xadupre** [comment]
@tmoreau89, I don't have permission on that branch.

**Comment by tmoreau89** [comment]
> @tmoreau89, I don't have permission on that branch.

Understood

**Comment by natke** [comment]
Apologies for the delay in merging @tmoreau89

**Pull request merged**